### PR TITLE
Add intergroup instructions to output of create

### PR DESCRIPTION
**Pull request by tpdownes**
- Replace intergroup warning with instructions to the user for executing export-outputs and import-inputs.
- Remove indentation from instructions to improve cut-and-pastability
- Update packer integration test to use export-outputs/import-inputs commands
- Modify image-builder example to adopt intergroup references

### Printed instructions for modified image-builder.yaml

```text
❯❯❯ ghpc create -w --vars project_id=test-project examples/image-builder.yaml
Terraform group 'builder-env' was successfully created in directory image-builder-001/builder-env
To deploy, run the following commands:

terraform -chdir=image-builder-001/builder-env init
terraform -chdir=image-builder-001/builder-env validate
terraform -chdir=image-builder-001/builder-env apply
ghpc export-outputs image-builder-001/builder-env

Packer group 'custom-image' was successfully created in directory image-builder-001/packer/custom-image
To deploy, run the following commands:

ghpc import-inputs image-builder-001/packer
cd image-builder-001/packer/custom-image
packer init .
packer validate .
packer build .
cd -

Terraform group 'cluster' was successfully created in directory image-builder-001/cluster
To deploy, run the following commands:

ghpc import-inputs image-builder-001/cluster
terraform -chdir=image-builder-001/cluster init
terraform -chdir=image-builder-001/cluster validate
terraform -chdir=image-builder-001/cluster apply

❯❯❯
```

### Old instructions (no intergroup references)

```text
❯❯❯ ghpc create -w --vars project_id=toolkit-demo-zero-e913 examples/image-builder.yaml
Terraform group 'builder-env' was successfully created in directory image-builder-001/builder-env
To deploy, run the following commands:
  terraform -chdir=image-builder-001/builder-env init
  terraform -chdir=image-builder-001/builder-env validate
  terraform -chdir=image-builder-001/builder-env apply

Packer group 'custom-image' was successfully created in directory image-builder-001/packer/custom-image
To deploy, run the following commands:
  cd image-builder-001/packer/custom-image
  packer init .
  packer validate .
  packer build .
  cd -

Terraform group 'cluster' was successfully created in directory image-builder-001/cluster
To deploy, run the following commands:
  terraform -chdir=image-builder-001/cluster init
  terraform -chdir=image-builder-001/cluster validate
  terraform -chdir=image-builder-001/cluster apply

❯❯❯
```

### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -193,64 +193,57 @@ For this example the following is needed in the selected region:
...
 
-1. Creates a network needed to build the image (see
-   [Custom Network](#custom-network-deployment-group-1)).
-2. Sets up a script that will be used to configure the image (see
-   [Toolkit Runners](#toolkit-runners-deployment-group-1)).
-3. Builds a new image by modifying the Slurm image (see
-   [Packer Template](#packer-template-deployment-group-2)).
-4. Deploys a Slurm cluster using the newly built image (see
-   [Slurm Cluster Based on Custom Image](#slurm-cluster-based-on-custom-image-deployment-group-3)).
+Create the deployment folder from the blueprint:
```
Comment (nick-stroud) [inline examples/README.md:208]: nit: In the updated version it is less obvious where instructions begin/end vs other material. Consider adding a minor header with something like "deployment instructions".

File: examples/image-builder.yaml
Comment (nick-stroud) [inline examples/image-builder.yaml:1]: We should also update the `hpc-cluster-intel-select.yaml` example. I am fine to do it separately but I think it should be done now and not pushed for later.

File: pkg/modulewriter/tfwriter.go [hunk 2]
```diff
@@ -331,14 +331,19 @@ func writeVersions(dst string) error {
...
-	if printIntergroupWarning {
-		fmt.Print(intergroupWarning)
+	fmt.Println()
+	if printImportInputs {
+		fmt.Printf("ghpc import-inputs %s\n", grpPath)
 	}
-	fmt.Printf("  terraform -chdir=%s init\n", grpPath)
-	fmt.Printf("  terraform -chdir=%s validate\n", grpPath)
-	fmt.Printf("  terraform -chdir=%s apply\n\n", grpPath)
+	fmt.Printf("terraform -chdir=%s init\n", grpPath)
```
Comment (nick-stroud) [inline pkg/modulewriter/tfwriter.go:340]: In reference to removing the indent: Can you put an example of how this change looks in the description. I think it is probably fine but am a little worried about readability. I would just like to validate that when it all comes together it is still clear.

Comment (tpdownes) [reply pkg/modulewriter/tfwriter.go:340]: Added. You'll observe that the indentation has been replaced by a line of vertical whitespace. Very nearly 100% of the time, I cut-and-paste the instructions from screen and having 2 columns of whitespace is a bit hard for "grabbing" the text as well making it harder to search my shell history for previous executions.

File: examples/README.md (hunk 1 above)
Comment (tpdownes) [reply examples/README.md:212]: Done with some improved text.

**Review by nick-stroud — APPROVED** [review]

**Pull request merged**